index.js

```diff
@@ -1,7 +1,7 @@
 var isFeed = require('ssb-ref').isFeed
 var DuplexPair = require('pull-pair/duplex')
 
-function DuplexError (message) {
+function ErrorDuplex (message) {
```

Comment:
I was confused by the name "DuplexError" because I thought this was a throwable error object, but it's really not, it's a Duplex stream that happens to emit errors from source and sink. So it's a "Duplex of Errors" hence "ErrorDuplex".